index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Products Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <header id="header">
+    <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" id="header-img" alt="site-logo">
+    <div id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </div>
+  </header>
+
+    <div id="container">
+      <div id="form-section">
+        <h2>Sign up for newletter</h2>
+        <form action="https://www.freecodecamp.com/email-submit" id="form">
+          <input type="email" id="email" name = "email" placeholder="Enter your email" required>
+          <br>
+          <input type="submit" id="submit" value="Submit"/>
+        </form>
+      </div>
+
+      <div id="how-it-works">
+        <iframe
+          id="video"
+          height="315"
+          src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0"
+          allowfullscreen
+        ></iframe>
+      </div>
+
+      <div id="features">
+        <div class="grid">
+          <div class="icon">
+            <i class="fa fa-3x fa-fire"></i></div>
+            <div class="desc">
+              <h2>Premium Materials</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally. This
+                will increase the longevity of your purchase.
+              </p>
+          </div>
+        </div>
+        <div class="grid">
+          <div class="icon">
+            <i class="fa fa-3x fa-truck"></i></div>
+          <div class="desc">
+            <h2>Fast Shipping</h2>
+            <p>
+              We make sure you recieve your trombone as soon as we have finished
+              making it. We also provide free returns if you are not satisfied.
+            </p>
+          </div>
+        </div>
+        <div class="grid">
+          <div class="icon">
+            <i class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+          </div>
+          <div class="desc">
+            <h2>Quality Assurance</h2>
+            <p>
+              For every purchase you make, we will ensure there are no damages or
+              faults and we will check and test the pitch of your instrument.
+            </p>
+          </div>
+        </div>
+      </div>
```

Comment:
Wrap this entire section in a ```<section>``` tag not a  ```<div>```

Reply:
The Same cases here:
- [ ]  I for better communication Instead of giving orders, make suggestions. So that the student will be fully motivated to implement the changes that you requested. for example instead of saying ` Wrap this entire section in a <section> tag not a <div> ` say `I think it would be better to wrap this entire section in a <section> tag not a <div>`.
- [ ]  I would encourage the use of a couple of emojis and/or gifs in all of your reviews. They make the code review more human-friendly :man: :woman:  not :robot: :alien: 
- [ ] I think your review lacked the `why` explanations on the necessity of the changes. I'd encourage you to always try and includes an explanation, a `why`. Thereby your review doesn't come off as commanding but rather complete and deliberate.